File: AppGenerateFiles/dbo/Model/Catalogo_Categoria.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Categoria : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_categoria { get; set; }
       public string? tipo { get; set; }
       public string? descripcion { get; set; }
       public int? plazo_limite { get; set; }
       public int? prioridad { get; set; }
       public bool? isEditable { get; set; }
       [OneToMany(TableName = "Detail_Prendas", KeyColumn = "id_categoria", ForeignKeyColumn = "id_categoria")]
       public List<Detail_Prendas>? Detail_Prendas { get; set; }
       [OneToMany(TableName = "Transactional_Valoracion", KeyColumn = "id_categoria", ForeignKeyColumn = "id_categoria")]
       public List<Transactional_Valoracion>? Transactional_Valoracion { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Categoria.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Detail_Prendas }  from './Detail_Prendas.js'
import { Transactional_Valoracion }  from './Transactional_Valoracion.js'
class Catalogo_Categoria extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_categoria;
   /**@type {String}*/ tipo;
   /**@type {String}*/ descripcion;
   /**@type {Number}*/ plazo_limite;
   /**@type {Number}*/ prioridad;
   /**@type {Boolean}*/ isEditable;
   /**@type {Array<Detail_Prendas>} OneToMany*/ Detail_Prendas;
   /**@type {Array<Transactional_Valoracion>} OneToMany*/ Transactional_Valoracion;
}
export { Catalogo_Categoria }
